### ISSUE TITLE
Refine Python API docs styling

### ISSUE THREAD

**Issue by justinvp** [pulumi/docs#1595]
The styles for the API docs could use another round of refinement: fixing font sizes, indentation, etc., so it is more obvious where you on the page and the hierarchy of things.

- [x] TypeScript/JavaScript (#1669)
- [ ] Python

**Comment by justinvp** [comment]
I made some very minor improvements in #1690 to Python, but it could use a couple more hours cleaning up the styling across the board. Going to move this out to M27 to finish up.

**Comment by bgdnlp** [comment]
I was wondering about the status of this issue.

The Python docs are hard to follow, at least for someone new to Pulumi. The styling on Typescript pages looks much better. 

It would also be nice if at least the nav bar on the right, the one that says "On this page", wuold be in a separate pane from the center column. That way it could be scrolled independently. Helps with jumping around in long pages.

**Comment by justinvp** [comment]
Closing this as a duplicate of https://github.com/pulumi/docs/issues/4615. We don't need this separate issue. We're tracking the work in https://github.com/pulumi/docs/issues/4615.
